### PR TITLE
Added Resources

### PR DESCRIPTION
**Pull request by PrabhanshuAttri**
Added CS50x, DevDocs, HTML from W3C, CheatSheet and Edx

### CODE REVIEW

File: learn_to_program.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 Hi! Feel free to contribute new links and resources to this document. Please add your resource under the appropriate heading, creating new ones where necessary. It'd also be great if you added a few words describing your resource! For example: if it's a paid resource, what format it's in, if it's particularly good for beginner or advanced people, etc.
 
+#####Offline Documentation
+*   [DevDocs](http://devdocs.io)
+
+#####Resources to start with programming
```
Comment (jig08) [inline learn_to_program.md]: Add this under the `General Programming` heading only, maybe ?

File: learn_to_program.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 Hi! Feel free to contribute new links and resources to this document. Please add your resource under the appropriate heading, creating new ones where necessary. It'd also be great if you added a few words describing your resource! For example: if it's a paid resource, what format it's in, if it's particularly good for beginner or advanced people, etc.
 
+#####Offline Documentation
+*   [DevDocs](http://devdocs.io)
```
Comment (jig08) [inline learn_to_program.md]: You could also put this under the `General Programming` heading with a special mention of `offline documentation` ...

**Comment by jig08** [comment]
Also @PrabhanshuAttri squash your commits :)

**Comment by PrabhanshuAttri** [comment]
I will keep in mind for future

**Comment by jig08** [comment]
Sure @PrabhanshuAttri .
Would you mind squashing the commits into a single one ? We would then be good to merge :)

**Comment by PrabhanshuAttri** [comment]
I committed these on web that's why I am unable to squash them. You can do it while merging also, I guess. [PR Squash and Merge](https://help.github.com/articles/about-pull-request-merge-squashing/)

**Comment by jig08** [comment]
@PrabhanshuAttri 
>This branch has conflicts that must be resolved

Kindly resolve 'em.

**Comment by jig08** [comment]
@PrabhanshuAttri Any updates on this ?

**Comment by PrabhanshuAttri** [comment]
Created a new PR. Had problems in pulling these commits and squashing them #48

**Pull request closed without merging**